dev/bots/test/analyze-sample-code_test.dart

```diff
@@ -14,7 +14,7 @@ void main() {
     );
     final List<String> stdoutLines = process.stdout.toString().split('\n');
     final List<String> stderrLines = process.stderr.toString().split('\n')
-      ..removeWhere((String line) => line.startsWith('Analyzer output:'));
+      ..removeWhere((String line) => line.startsWith('Analyzer output:') || line.startsWith('Building flutter tool...'));
```

Comment:
oh, i didn't anticipate that we'd be grepping stderr

Reply:
Yeah, I need to dig in a bit and fix these tests anyway

Reply:
there be dragons under `//dev/`...